plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Help.js

```diff
@@ -142,7 +142,7 @@ Ext.define('NX.controller.Help', {
    */
   onFeatureHelp: function() {
     var me = this,
-        keyword = me.selectedFeature.get('helpKeyword'),
+        keyword = me.selectedFeature.get('helpKeyword').replace(new RegExp('_', 'g'), '%20'),
```

Comment:
This is bascially a replaceAll .. wasnt sure if you want to use some sort of function you already got so I just made it work in straight JS. Could also add some sort of comment if you want that in the JS files.